maintenance/enabling-bpf.md

```diff
@@ -264,7 +281,8 @@ To enable eBPF mode, change Felix configuration parameter  `BPFEnabled` to `true
 calicoctl patch felixconfiguration default --patch='{"spec": {"bpfEnabled": true}}'
 ```
 
-Enabling eBPF node can disrupt existing workload connections.  After enabling eBPF mode you may need to restart workload pods in order for them to restart connections. 
+Enabling eBPF node can disrupt existing workload connections.  After enabling eBPF mode you may need to restart workload pods in order for them to restart connections.  In particular, it's a good idea to restart `kube-dns`
```

Comment:
Maybe just put the kube-dns restart instruction in as a command?  Its easy to skim read this and miss the "its a good idea" note?